Extras/GIMPACTUtils/btGImpactConvexDecompositionShape.h

```diff
@@ -1,5 +1,5 @@
 /*! \file btGImpactConvexDecompositionShape.h
-\author Francisco Le�n N�jera
+\author Francisco Leon Najera
```

Comment:
ó and á are [unicode](http://www.fileformat.info/info/unicode/char/00f3/index.htm) [points](http://www.fileformat.info/info/unicode/char/00e1/index.htm)

Reply:
> I normalized Francisco León Nájera to Francisco Leon Najera as was already done in most other source files they authored, but if preferred I could also reencode it as Francisco León Nájera in UTF-8.